fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -1389,4 +1389,33 @@ public TransactionState getTransactionStateByCallbackId(long callbackId) {
         }
         return null;
     }
+
+    public List<Long> getTransactionIdByCoordinateBe(String coordinate, int limit) {
```

Comment:
Would better rename `coordinate` to `coordinateHost`

Reply:
done